.github/workflows/pull-request.yml

```diff
@@ -15,7 +15,8 @@ jobs:
           registry-url: https://registry.npmjs.org/
 
       - name: Install dependencies
-        run: npm install
+        # Using --no-optional to reduce vulnerabilities
+        run: npm install --no-optional
```

Comment:
Is there a particular reason why we do not use `npm ci` here?

Reply:
Good point, I will change to be `npm ci --no-optional`